src/openassetio-python/tests/package/pluginSystem/test_hybridpluginsystemmanagerimplementationfactory.py

```diff
@@ -0,0 +1,1637 @@
+
+        name = hybrid_manager_interface.displayName()
+
+        assert name == "B"
+
+        manager_interface = HybridPluginSystemManagerImplementationFactory(
+            [factory_b, factory_a], mock_logger
+        ).instantiate(the_plugin_identifier)
+
+        name = manager_interface.displayName()
+
+        assert name == "C"
+
+    def test_info_aggregates_all_implementations_preferring_first(
+        self,
+        manager_interface_a,
+        manager_interface_b,
+        hybrid_manager_interface,
+    ):
+        manager_interface_a.mock.info.return_value = {"x": 1, "y": "i"}
+        manager_interface_b.mock.info.return_value = {"y": "k", "z": 1.2}
+
+        assert hybrid_manager_interface.info() == {"x": 1, "y": "i", "z": 1.2}
+
+    def test_settings_aggregates_all_implementations_preferring_first(
+        self,
+        a_host_session,
+        manager_interface_a,
+        manager_interface_b,
+        hybrid_manager_interface,
+    ):
+        manager_interface_a.mock.settings.return_value = {"x": "j", "y": "i"}
+        manager_interface_b.mock.settings.return_value = {"y": "k", "z": 1.2}
+
+        assert hybrid_manager_interface.settings(a_host_session) == {"x": "j", "y": "i", "z": 1.2}
+
+    @pytest.mark.parametrize(
+        "capability",
+        # Couple of arbitrary capabilities.
+        (
+            ManagerInterface.Capability.kPublishing,
+            ManagerInterface.Capability.kEntityReferenceIdentification,
+        ),
+    )
+    @pytest.mark.parametrize(
+        "hasCapability_a,hasCapability_b", itertools.product((True, False), repeat=2)
+    )
+    def test_hasCapability_is_aggregated_from_all_child_implementations(
+        self,
+        capability,
+        hasCapability_a,
+        hasCapability_b,
+        the_plugin_identifier,
+        manager_interface_a,
+        manager_interface_b,
+        hybrid_factory,
+    ):
+        manager_interface_a.mock.hasCapability.side_effect = (
+            lambda cap: cap == capability and hasCapability_a
+        )
+        manager_interface_b.mock.hasCapability.side_effect = (
+            lambda cap: cap == capability and hasCapability_b
+        )
+
+        manager_interface = hybrid_factory.instantiate(the_plugin_identifier)
+
+        assert manager_interface.hasCapability(capability) is (hasCapability_a or hasCapability_b)
+
+    def test_initialize_initializes_all_child_implementations_forwarding_all_arguments(
+        self, manager_interface_a, manager_interface_b, hybrid_manager_interface, a_host_session
+    ):
+        expected_settings = {"x": 1, "y": "2", "z": False}
+
+        hybrid_manager_interface.initialize(expected_settings, a_host_session)
+
+        manager_interface_a.mock.initialize.assert_called_once_with(
+            expected_settings, a_host_session
+        )
+        manager_interface_b.mock.initialize.assert_called_once_with(
+            expected_settings, a_host_session
+        )
```

Comment:
Having a fixture called `a_host_session` was somewhat confusing here, as I thought it should've been `b_host_session` for the second assert.

If it's intentional that the a vs b is a suffix, rather than a_host_session is regular English as a prefix, then that's something for me to watch for.

Reply:
:grin:  This was a style that Tom introduced, and I quite like.  I.e. fixtures should have a prefix like `a_` or `an_` or `the_` to make it clear they're generic representatives.  That style is used a lot in the codebase, but not rigorously consistently.

Reply:
Just something to get used to I imagine.

I was thinking `the_host_session` may have been useful, as there's only one host ever?

Reply:
Good point, `the_` probably does make more sense for the `HostSession` instance.  Unfortunately it's a shared (`conftest.py`) fixture with many other tests, so renaming it would mean a _lot_ of other tests being changed too - a bit out of scope for this PR.

Reply:
Backlog suggestion then :)

Reply:
Added https://github.com/OpenAssetIO/OpenAssetIO/issues/1379 to track this.